src/main/java/net/tropicraft/core/common/data/TropicraftLootTableProvider.java

```diff
@@ -63,6 +83,103 @@ protected void validate(Map<ResourceLocation, LootTable> map, ValidationTracker
         // Nothing for now, but chest loot tables eventually
     }
     
+    private static class Entities extends EntityLootTables {
+    	@Override
+    	protected void addTables() {
+
+    		registerCreeperLoot();
+			registerEntityLoot(TropicraftEntities.IGUANA, TropicraftItems.IGUANA_LEATHER, TropicraftItems.SCALE, ConstantRange.of(3));
+			registerMinimalLootTable(TropicraftEntities.TROPI_SKELLY);
+			registerEntityLoot(TropicraftEntities.EIH, TropicraftBlocks.CHUNK.get().asItem(), ConstantRange.of(3));
+			registerEntityLoot(TropicraftEntities.SEA_TURTLE, TropicraftItems.TURTLE_SHELL);
+			registerEntityLoot(TropicraftEntities.MARLIN, TropicraftItems.FRESH_MARLIN, RandomValueRange.of(1, 3));
+			registerMinimalLootTable(TropicraftEntities.FAILGULL);
+			registerEntityLoot(TropicraftEntities.DOLPHIN, TropicraftItems.TROPICAL_FERTILIZER, RandomValueRange.of(1, 3));
+			registerMinimalLootTable(TropicraftEntities.SEAHORSE);
+			registerFrogLoot();
+			registerEntityLoot(TropicraftEntities.SEA_URCHIN, TropicraftItems.SEA_URCHIN_ROE);
+			registerMinimalLootTable(TropicraftEntities.SEA_URCHIN_EGG_ENTITY);
+			registerEntityLoot(TropicraftEntities.STARFISH, TropicraftItems.STARFISH);
+			registerMinimalLootTable(TropicraftEntities.STARFISH_EGG);
+			registerMinimalLootTable(TropicraftEntities.V_MONKEY);
+			registerMinimalLootTable(TropicraftEntities.RIVER_SARDINE);
+			registerEntityLoot(TropicraftEntities.RIVER_SARDINE, TropicraftItems.RAW_FISH);
+			registerEntityLoot(TropicraftEntities.PIRANHA, TropicraftItems.RAW_FISH);
+			registerEntityLoot(TropicraftEntities.TROPICAL_FISH, TropicraftItems.RAW_FISH);
+			registerEntityLoot(TropicraftEntities.EAGLE_RAY, TropicraftItems.RAW_RAY);
+			registerMinimalLootTable(TropicraftEntities.TROPI_SPIDER);
+			registerMinimalLootTable(TropicraftEntities.TROPI_SPIDER_EGG);
+			registerMinimalLootTable(TropicraftEntities.ASHEN);
+			registerEntityLoot(TropicraftEntities.HAMMERHEAD, TropicraftItems.TROPICAL_FERTILIZER, RandomValueRange.of(1, 3));
+			registerMinimalLootTable(TropicraftEntities.SEA_TURTLE_EGG);
+			registerMinimalLootTable(TropicraftEntities.TROPI_BEE);
+			registerMinimalLootTable(TropicraftEntities.COWKTAIL);
+			registerEntityLoot(TropicraftEntities.MAN_O_WAR, Items.SLIME_BALL, RandomValueRange.of(3, 4));
+}
+
+		//Only Green frogs should drop poison skins!
+    	private static final ILootCondition.IBuilder GREEN = 
+    			EntityHasProperty.builder(LootContext.EntityTarget.THIS, 
+    					EntityPredicate.Builder.create().flags(
+    							new EntityFlagsPredicate(null, null, null, null, false)).nbt(
+    									new NBTPredicate(Util.make(new CompoundNBT(), c -> c.putInt("Type", TreeFrogEntity.Type.GREEN.ordinal())))));
```

Comment:
The indentation nesting here is a bit excessive
```suggestion
    	private static final ILootCondition.IBuilder GREEN = 
    			EntityHasProperty.builder(LootContext.EntityTarget.THIS, EntityPredicate.Builder.create()
    			    .flags(new EntityFlagsPredicate(null, null, null, null, false))
    			    .nbt(new NBTPredicate(Util.make(new CompoundNBT(), c -> c.putInt("Type", TreeFrogEntity.Type.GREEN.ordinal())))));
```